src/module-elasticsuite-tracker/Model/ResourceModel/EventQueue.php

```diff
@@ -79,7 +79,12 @@ public function getEvents($limit = null)
         $eventData = $this->getConnection()->fetchAll($select);
 
         foreach ($eventData as &$currentEvent) {
```

Comment:
Due to the foreach syntax here, it's not possible to simply "unset" and bypass the whole event in the catch.